### PR TITLE
added redhat(yumrepo) support

### PR DESCRIPTION
**Pull request by ume3**
## background
- Please support this osfamily
 - redhat (centos)

### Check CentOS7.2(1511) / Redhat7.1
refs
 - https://td-agent-package-browser.herokuapp.com/2/redhat/
 - http://docs.puppetlabs.com/puppet/latest/reference/type.html#yumrepo

### vagrant test environment
- https://vagrantcloud.com/puppetlabs/boxes/centos-7.2-64-puppet

### CODE REVIEW

**Comment by ume3** [comment]
oh .. Travis CI .
sorry not test yet  ...

**Comment by Moozaliny** [comment]
Thanks for the PR, very much appreciated! 
Looks like something changed on Travis CI, your changes shouldn't have broken anything. We are working on fixing this and then will get back to your PR.

File: manifests/repo/yum.pp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+class fluentd::repo::yum (
+  $descr    = 'TreasureData',
+  $baseurl  = 'https://packages.treasuredata.com/2/redhat/$releasever/$basearch',
+  $enabled  = '1',
+  $gpgcheck = '1',
+  $gpgkey   = 'https://packages.treasuredata.com/GPG-KEY-td-agent',
+) {
+
+  include '::yumrepo'
```
Comment (Moozaliny) [inline manifests/repo/yum.pp]: is this really needed? 
yumrepo is one of puppets built-in types and doesn't need an extra module afaik

File: manifests/repo/yum.pp [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Configure yumrepo
+#
+class fluentd::repo::yum (
+  $descr    = 'TreasureData',
+  $baseurl  = 'https://packages.treasuredata.com/2/redhat/$releasever/$basearch',
+  $enabled  = '1',
+  $gpgcheck = '1',
+  $gpgkey   = 'https://packages.treasuredata.com/GPG-KEY-td-agent',
```
Comment (Moozaliny) [inline manifests/repo/yum.pp]: would you may be so kind and add an $ensure parameter

File: manifests/repo/yum.pp (hunk 1 above)
Comment (ume3) [inline manifests/repo/yum.pp]: thanks for review
yes, not really needed
I will fix

File: manifests/repo/yum.pp (hunk 2 above)
Comment (ume3) [inline manifests/repo/yum.pp]: I also think that it is a good
i will add `$ensure parameter`

**Comment by ume3** [comment]
thanks for review
i did it

Continue Thank you

**Pull request merged**